.github/workflows/CI-build.yml

```diff
@@ -36,12 +36,12 @@ jobs:
         with:
           args: unzip -qq ./p2/target/p2-1.0.0-SNAPSHOT.zip -d ./p2/target/p2-1.0.0-SNAPSHOT
           
-      - name: Push p2 build to another branch
-        uses: s0/git-publish-subdir-action@master
-        env:
-          REPO: self
-          BRANCH: p2-repository # The branch name where you want to push the assets
-          FOLDER: ./p2/target/p2-1.0.0-SNAPSHOT # The directory where your assets are generated
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # GitHub will automatically add this - you don't need to bother getting a token
-          MESSAGE: "Build: plugins available from p2-repository" # The commit message
+      #- name: Push p2 build to another branch
+        #uses: s0/git-publish-subdir-action@master
+        #env:
+          #REPO: self
+          #BRANCH: p2-repository # The branch name where you want to push the assets
+          #FOLDER: ./p2/target/p2-1.0.0-SNAPSHOT # The directory where your assets are generated
+          #GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # GitHub will automatically add this - you don't need to bother getting a token
+          #MESSAGE: "Build: plugins available from p2-repository" # The commit message
```

Comment:
disable ?

Reply:
The p2-1.0.0-SNAPSHOT cannot be pushed to p2-repository,
There is this message:
remote: Permission to orbisgis/orbisrcp.git denied to github-actions[bot].
fatal: unable to access 'https://github.com/orbisgis/orbisrcp.git/': The requested URL returned error: 403

Reply:
@SPalominos  could you check when you will have the time ;-)

Reply:
Is the GITHUB_TOKEN already active ?